src/components/cardbuilder/card.scss

```diff
@@ -389,6 +389,10 @@ button::-moz-focus-inner {
     transform: translate(-50%, -50%);
 }
 
+.cardImageContainer .cardImageIcon {
+    margin: auto; /* 'justify-content: center' doesn't work in Safari 10 */
```

Comment:
I wondered why this was broken, but not enough to actually dig into it. :sweat_smile: